sdk/storage/azure-storage-queue/tests/test_queue.py

```diff
@@ -120,49 +131,59 @@ def test_create_queue_with_options(self):
         self.assertEqual('test', props.metadata['val1'])
         self.assertEqual('blah', props.metadata['val2'])
 
-    @record
+    @ResourceGroupPreparer()     
+    @StorageAccountPreparer(name_prefix='pyacrstorage', playback_fake_resource=FAKE_STORAGE)
     def test_delete_non_existing_queue(self):
         # Action
-        queue_client = self._get_queue_reference()
+        qsc = QueueServiceClient(self._account_url(storage_account.name), storage_account_key)
+        queue_client = self._get_queue_reference(qsc)
 
         # Asserts
         with self.assertRaises(ResourceNotFoundError):
             queue_client.delete_queue()
 
-    @record
-    def test_delete_existing_queue_fail_not_exist(self):
+    @ResourceGroupPreparer()     
+    @StorageAccountPreparer(name_prefix='pyacrstorage', playback_fake_resource=FAKE_STORAGE)
+    def test_delete_existing_queue_fail_not_exist(self, resource_group, location, storage_account, storage_account_key):
         # Action
-        queue_client = self._get_queue_reference()
+        qsc = QueueServiceClient(self._account_url(storage_account.name), storage_account_key)
+        queue_client = self._get_queue_reference(qsc)
 
         created = queue_client.create_queue()
         deleted = queue_client.delete_queue()
 
         # Asserts
         self.assertIsNone(deleted)
 
-    @record
-    def test_list_queues(self):
+    @ResourceGroupPreparer()    
+    @StorageAccountPreparer(name_prefix='pyacrstorage', playback_fake_resource=FAKE_STORAGE)
+    def test_list_queues(self, resource_group, location, storage_account, storage_account_key):
         # Action
-        queues = list(self.qsc.list_queues())
+        qsc = QueueServiceClient(self._account_url(storage_account.name), storage_account_key)
+        queue_client = self._get_queue_reference(qsc)
+        queue_client.create_queue()
+        queues = list(qsc.list_queues())
 
         # Asserts
         self.assertIsNotNone(queues)
-        self.assertTrue(len(self.test_queues) <= len(queues))
+        self.assertEqual(len(queues), 1)
```

Comment:
Now that we're no longer unsing unittest, we need to move away from the `self.assert` syntax, e.g.:
assertTrue, assertEqual, assertIsNone, assertIsNotNone, assertFalse etc etc

In most cases we should be able to use just a basic assert statement:
```
assert len(queues) == 1
```

Reply:
I'll script this across blob, queues and files onve i have green CIs for the other two. manually refactoring every assert statement doesn't feel very practical.